msg/HydrophoneDeltas.msg

```diff
@@ -0,0 +1,4 @@
+std_msgs/Header header
+duration t1
```

Comment:
I feel like these could be named to better represent which hydrophone deltas they are. (e.g. xDelta, yDelta, zDelta) I'm also assuming you're using these as the time deltas from the reference hydrophone, correct?